plans/cd4pe_job.pp

```diff
@@ -2,20 +2,23 @@
   TargetSpec                      $targets,
   String[1]                       $job_instance_id,
   String[1]                       $cd4pe_web_ui_endpoint,
-  String[1]                       $cd4pe_token,
+  Sensitive[String[1]]            $cd4pe_token,
   String[1]                       $cd4pe_job_owner,
   Optional[Array[String[1]]]      $env_vars = undef,
   Optional[String[1]]             $docker_image = undef,
   Optional[Array[String[1]]]      $docker_run_args = undef,
   Optional[String[1]]             $base_64_ca_cert = undef,
 
 ) {
+
+  $token = Sensitive($cd4pe_token)
+
   return run_task(
     'cd4pe_jobs::run_cd4pe_job',
     $targets,
     'job_instance_id' => $job_instance_id,
     'cd4pe_web_ui_endpoint' => $cd4pe_web_ui_endpoint,
-    'cd4pe_token' => $cd4pe_token,
+    'cd4pe_token' => $token.unwrap,
```

Comment:
idk how sensitive params work, but don't you mean `$cd4pe_token.unwrap` ?

Reply:
we resolved this in Slack, `$token` is assigned above.

i wrote this following the official docs: https://puppet.com/docs/bolt/latest/writing_plans.html